### ISSUE TITLE
Training Loss

### ISSUE THREAD

**Issue by ooooohira** [whwu95/FreeVA#8]
Thanks for Great Work!!!

If you can, please share training loss when training model with Video-ChatGPT dataset.

I implemented a reimplementation, and after training a 4-frame model for 3 epochs, the loss converged to around 3.

Is this considered high?
